### PR TITLE
fix: wrong tooltip in radar chart in firefox

### PR DESCRIPTION
**Pull request by jyothishjose6190**
## Description
In Firefox alone, wrong tooltip was shown for radar chart

- **on a bugfix**: Bug: Wrong tooltip is showing for radar chart #748. For firefox, radialAxisData was not sorted on the basis of axisRadian

### Testing
Run all test cases using npm run lint && npm run test

### Checklist:
- [x] My changes generate no new warnings

### CODE REVIEW

File: projects/observability/src/shared/components/radar/tooltip/radial-data-lookup-strategy.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ export class RadialDataLookupStrategy {
 
   public constructor(private readonly allSeries: RadarSeries[], radialAxisData: RadarAxisData[]) {
     this.radialBisector = bisector(axisData => axisData.axisRadian);
-    this.radialAxisData = clone(radialAxisData);
+    this.radialAxisData = clone(radialAxisData.sort(axisData => axisData.axisRadian));
```
Comment (aaron-steinfeld) [inline projects/observability/src/shared/components/radar/tooltip/radial-data-lookup-strategy.ts:18]: please sort the cloned array instead of the argument

Comment (aaron-steinfeld) [reply projects/observability/src/shared/components/radar/tooltip/radial-data-lookup-strategy.ts:18]: Out of curiosity, if this data is not sorted why is this not an issue in other browsers?

Comment (jyothishjose6190) [reply projects/observability/src/shared/components/radar/tooltip/radial-data-lookup-strategy.ts:18]: It is sorted in other browsers like chrome and safari.
Without the fix, in chrome radialAxisData, is sorted by axisRadian
Screenshot 2021-07-15 at 10 51 29 AM


The same code in firefox returns  radialAxisData is not sorted
Screenshot 2021-07-15 at 10 52 19 AM

File: projects/observability/src/shared/components/radar/tooltip/radial-data-lookup-strategy.ts [hunk 2]
```diff
@@ -16,6 +16,7 @@ export class RadialDataLookupStrategy {
   public constructor(private readonly allSeries: RadarSeries[], radialAxisData: RadarAxisData[]) {
     this.radialBisector = bisector(axisData => axisData.axisRadian);
     this.radialAxisData = clone(radialAxisData);
+    this.radialAxisData = this.radialAxisData.sort(axisData => axisData.axisRadian);
```
Comment (itssharmasandeep) [inline projects/observability/src/shared/components/radar/tooltip/radial-data-lookup-strategy.ts:19]: We can do this in line. Just doing the sorting after cloning.
`this.radialAxisData = clone(radialAxisData).sort(axisData => axisData.axisRadian)`

File: projects/observability/src/shared/components/radar/tooltip/radial-data-lookup-strategy.ts [hunk 3]
```diff
@@ -41,6 +42,7 @@ export class RadialDataLookupStrategy {
     const leftValue = this.radialAxisData[insertionIndex - 1] as RadarAxisData | undefined;
     const rightValue = this.radialAxisData[insertionIndex] as RadarAxisData | undefined;
 
+
```
Comment (itssharmasandeep) [inline projects/observability/src/shared/components/radar/tooltip/radial-data-lookup-strategy.ts:45]: Please remove extra line!!

File: projects/observability/src/shared/components/radar/tooltip/radial-data-lookup-strategy.ts (hunk 1 above)
Comment (aaron-steinfeld) [reply projects/observability/src/shared/components/radar/tooltip/radial-data-lookup-strategy.ts]: oh strange - thanks for explaining!

**Review by aaron-steinfeld — APPROVED** [review]

**Comment by aaron-steinfeld** [comment]
Welcome and thanks @jyothishjose6190 !

**Comment by jyothishjose6190** [comment]
> Welcome and thanks @jyothishjose6190 !

thank you

**Pull request merged**